packages/widget/src/index.tsx

```diff
@@ -150,7 +161,9 @@ export const Widget: React.FunctionComponent<Props> = (props) => {
   }, [chat.conversationHandler]);
 
   // Expanded state
-
+  useEffect(() => {
+    setExpanded(Boolean(props.initiallyExpanded));
+  }, [Boolean(props.initiallyExpanded)])
```

Comment:
I have implementation clarity concerns here per https://beta.reactjs.org/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state.